src/ui/component/skeleton/skeleton.scss

```diff
@@ -0,0 +1,44 @@
+@import '@/ui/style/theme.scss';
+
+.okp4-dataverse-portal-skeleton {
+  @include with-theme {
+    background-color: rgba(themed('skeleton-base'), 0.11);
+    position: relative;
+    overflow: hidden;
+    border-radius: 10px;
+
+    &::before {
+      content: '\00a0';
+    }
+
+    &::after {
+      animation: animation-translateX 1.3s linear 0.5s infinite;
+      background: linear-gradient(90deg, transparent, rgba(themed('skeleton-animation'), 0.04), transparent);
```

Comment:
Could you add this gradient to the `palette.scss` ?

Reply:
this gradient uses a `themed` value, is it still possible to add it to the palette when it depends on a mixin?